Mudz.Common/Domain/Environment/Map/Grid.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Mudz.Common.Domain.Environment.Map.Room;
```

Comment:
Had no intention of adding any files, so no idea why this is here. Weird working copy perhaps?